File: resources/templates/javascript/stub/http_handler.js
var RequestToBeSentAction = Packages.burp.api.montoya.http.handler.RequestToBeSentAction;
var ResponseReceivedAction = Packages.burp.api.montoya.http.handler.ResponseReceivedAction;
var HttpRequestToBeSent = Packages.burp.api.montoya.http.handler.HttpRequestToBeSent;
var HttpResponseReceived = Packages.burp.api.montoya.http.handler.HttpResponseReceived;
function handleHttpRequestToBeSent(montoyaApi, httpRequestToBeSent) {
    return httpRequestToBeSent.withAddedHeader("DemoHttpHandler","Added by user defined script");
}

function handleHttpResponseReceived(montoyaApi, httpResponseReceived) {
    return null;
}

File: resources/templates/jython/stub/http_handler.py
from burp.api.montoya.http.handler import RequestToBeSentAction
from burp.api.montoya.http.handler import ResponseReceivedAction
from burp.api.montoya.http.handler import HttpRequestToBeSent
from burp.api.montoya.http.handler import HttpResponseReceived
def handleHttpRequestToBeSent(montoyaApi, httpRequestToBeSent):
    return httpRequestToBeSent.withAddedHeader("DemoHttpHandler","Added by user defined script")

def handleHttpResponseReceived(montoyaApi, httpResponseReceived):
    return None